src/AzSK.ADO/Framework/BugLog/BugMetaInfoProvider.ps1

```diff
@@ -141,13 +150,14 @@ class BugMetaInfoProvider {
                             }
                         }
                         # if no build/release jobs associated with service connection, then fecth assignee from permissions
-                        else 
+                        # asignee not found from build/release
+                        elseif (!$assignee) 
                         {
                             try {
                                 $projectId = ($ControlResult.ResourceContext.ResourceId -split "project/")[-1].Split('/')[0]
                                 $apiURL = "https://dev.azure.com/{0}/_apis/securityroles/scopes/distributedtask.serviceendpointrole/roleassignments/resources/{1}_{2}" -f $organizationName, $projectId, $ControlResult.ResourceContext.ResourceId.split('/')[-1]
                                 $responseObj = [WebRequestHelper]::InvokeGetWebRequest($apiURL);
-                                $roles =   @($responseObj | where {$_.role.displayname -eq 'Administrator'} |select identity) | where {(-not ($_.identity.displayname).Contains("\")) -and ($_.identity.displayname -notin @("GitHub", "Microsoft.VisualStudio.Services.TFS"))}
+                                $roles =   @(($responseObj | where {$_.role.displayname -eq 'Administrator'} |select identity) | where {(-not ($_.identity.displayname).Contains("\")) -and ($_.identity.displayname -notin @("GitHub", "Microsoft.VisualStudio.Services.TFS"))} )
                                 if ($roles.Count -gt 0) {
                                     $userId = $roles[0].identity.id
                                     $assignee = $this.getUserFromUserId($organizationName, $userId)
```

Comment:
should we check in getUserFromUserId at the end if user is active?

Reply:
Checking latter on outside in Autobuglog class